web/src/inspector.js

```diff
@@ -783,6 +844,72 @@ export class Inspector extends EventTarget {
     list.append(...rows);
   }
 
+  ///  SETTINGS PAGE
+
+  _onTorrentSettingsChanged(event_) {
+    if (this.updating) {
+      return;
+    }
+
+    const { key } = event_.target.dataset;
+    const { torrents, controller } = this;
+    const val = Inspector._getValue(event_.target);
+
+    if (val === '') {
+      return;
+    }
+
+    event_.target.lastvalue = val;
```

Comment:
_updateSettings is called periodically by the automatic refresh timer, and if we allow all updates to push new values to editable fields, it will tend to overwrite new values as a user is typing into them.

The 'changed' event (which this handler is bound to) fires when a user "commits" a change to a field. `lastvalue` stores the most recent committed value, and new values pushed to the UI will be ignored if they match this value, to prevent editing out from under someone who's typing.